crates/holochain/src/local_network_tests.rs

```diff
@@ -23,9 +23,14 @@ async fn conductors_call_remote(num_conductors: usize) {
         .map(|c| c.into_cells().into_iter().next().unwrap())
         .collect();
 
-    // Make sure that genesis records are integrated now that conductors have discovered each other. This makes it
-    // more likely that Kitsune knows about all the agents in the network to be able to make remote calls to them.
-    await_consistency(60, cells.iter()).await.unwrap();
```

Comment:
This does little to ensure we're actually connected to everyone. It just means each conductor connected to someone and got ops.